.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Do you actually need this? I do regularly attach vscode debugger to my unit tests without having a launch.json file.

Reply:
vscode always tells I need one 🤔

Reply:
interesting. Well, won't really hurt